.github/workflows/create-hotfix-branch.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
 
     # Only allow these users to create new hotfix branch from 'main'
```

Comment:
do it based on team membership instead - https://github.com/rudderlabs/rudder-auth/pull/410/commits/01c66c2532656f81cae1702d9e7b960a7f97743f